test.js

```diff
@@ -8,6 +8,78 @@ test('assign own enumerable propreties from source to target object', t => {
 	t.end();
 });
 
+test('do not assign null values', t => {
+	t.same(fn({}, {foo: null}), {});
+	t.same(fn({foo: null}, {foo: null}), {foo: null});
+	t.end();
+});
+
+test('assign values to null targets', t => {
+	t.same(fn({foo: null}, {foo: {}}), {foo: {}});
+	t.end();
+});
+
+test('do not assign undefined values', t => {
+	t.same(fn({}, {foo: undefined}), {});
```

Comment:
That is also true, because `Object.assign` is not recursive:
```js
Object.assign(x, undefined) === x
```
i.e. `undefined` (and `null`) values are ignored.